### PR TITLE
chore!: optimize `getTransactions` query

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Relates to #3289

# Release notes

In this release, we:

- Optimize graphQL query for `Provider.getTransactions`

# Summary

This PR updates the [getTransactions](https://github.com/FuelLabs/fuels-ts/blob/9ebd6c224482a9b3c409409e9f0e14cb0de38224/packages/account/src/providers/operations.graphql#L481) query to return only the `rawPayload` property

The [Provider.getTransactions](https://github.com/FuelLabs/fuels-ts/blob/9ebd6c224482a9b3c409409e9f0e14cb0de38224/packages/account/src/providers/provider.ts#L1685) method, which uses this query, only requires the `rawPayload` property. Therefore, the broader `transactionFragment`, which includes unnecessary data, is no longer used.

# Breaking Changes

The response format for `Provider.getTransactions` remains the same. However, the response format for the query `Provider.operations.getTransactions` has been modified.

```graphql
// before
query getTransactions {
  id
  rawPayload
  status {
    ...
  }
}
```

```graphql
// after
query getTransactions {
  rawPayload
}
```

# Checklist

- [x] All **changes** are **covered** by **tests** (or not applicable)
- [x] All **changes** are **documented** (or not applicable)
- [x] I **reviewed** the **entire PR** myself (preferably, on GH UI)
- [x] I **described** all **Breaking Changes** (or there's none)

### CODE REVIEW

File: .changeset/nervous-impalas-serve.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
```
Comment (maschad) [inline .changeset/nervous-impalas-serve.md:2]:
```
"@fuel-ts/account": minor
```

Isn't this a breaking change given we [changing](https://github.com/FuelLabs/fuels-ts/pull/3336/files#diff-df8d51395ea0431bdaeb7f953b65d98fc4fc0a4f2cce253f31a2b4c87c616c14R495) the response of `getTransactions` to return two less fields? i.e. the `id` and `transactionStatusFragment` ?

**Review by maschad — CHANGES_REQUESTED** [review]

File: .changeset/nervous-impalas-serve.md (hunk 1 above)
Comment (Torres-ssf) [reply .changeset/nervous-impalas-serve.md:2]: But @maschad The [Provider.getTransactions](https://github.com/FuelLabs/fuels-ts/blob/9ebd6c224482a9b3c409409e9f0e14cb0de38224/packages/account/src/providers/provider.ts#L1685) has exactly the same response, not property was removed from it.

Comment (maschad) [reply .changeset/nervous-impalas-serve.md:2]: I agree with you but if someone called `provider.operations.getTransactions()` the response would change and thus it's breaking.

Comment (danielbate) [reply .changeset/nervous-impalas-serve.md:2]: We made #3286 and #3296 breaking for the same reason.

**Review by danielbate — COMMENTED** [review]
LGTM apart from the changeset 👍🏻

File: .changeset/nervous-impalas-serve.md (hunk 1 above)
Comment (Torres-ssf) [reply .changeset/nervous-impalas-serve.md:2]: @maschad Oook I get you now.

I think this one went unnoticed https://github.com/FuelLabs/fuels-ts/pull/3309

Comment (Torres-ssf) [reply .changeset/nervous-impalas-serve.md]: https://github.com/FuelLabs/fuels-ts/pull/3336/commits/fe146700202b4f338b73865c89b8abbe39552f75

**Review by petertonysmith94 — APPROVED** [review]
Could we add a migration suggestion to the breaking changes? Maybe just a suggestion to user `Provider.getTransactions`?

**Review by Dhaiwat10 — APPROVED** [review]

**Review by maschad — APPROVED** [review]

**Pull request merged**